tests/generation/test_generate.py

```diff
@@ -59,3 +59,93 @@ def test_model_generation_input_dimensions(neuron_model_path):
     # Using an incompatible input length
     with pytest.raises(ValueError, match="The input sequence length"):
         _test_model_generation(model, tokenizer, model.batch_size, input_length=model.max_length * 2)
+
+
+@is_inferentia_test
+@requires_neuronx
+def test_seq2seq_generation_beam(neuron_seq2seq_beam_path):
```

Comment:
It is a minor detail, but you could have parametrized this test with something like `"param, value", [["max_length",5], ...]`. I think I did that in my tests.